### DISCUSSION TITLE
Spiral / Debian 11 : emoji fonts broken?

### DISCUSSION THREAD

**Discussion by kr428** [SpiralLinux/SpiralLinux-project#220] [Q&A] [answered]
Folks; started sing SpiralLinux a while ago and so far really enjoy what I see. One minor yet somewhat annoying issue however, it seems that somehow emoji fonts are broken. By default, I'm unsure which emoj font is installed, but in most GTK applications, it just shows monochrome emojis with a few dedicated icons (like 🤔) generally missing:

![image](https://user-images.githubusercontent.com/685879/211906778-9df4dec9-3ccb-420e-bc8c-3a139a4c7a63.png)

On other Linux installs, I use fonts-noto-color-emoji for that purpose, but installing this on Spiral Linux makes things considerably worse - as in none of the GTK emoji selectors displaying _any_ emoji anymore, at all.

This is weird. Followed different guides on how to "fix" fontcache to make this work but none of them seemed to achieve anything meaningful. So, asking: Anyone else seen (and ideally) solved this? Any idea where to peek to get rid of that?
Thanks in advance and best,
Kristian

**Comment by geckolinux** [comment, marked as answer]
Hmm, thanks for reporting the issue.

What desktop environment are you using?

It looks like it's an upstream Debian issue. Could you please try these workarounds?
- https://unix.stackexchange.com/questions/692376/how-to-let-kde-on-debian-show-emojis
- https://bugs.freedesktop.org/show_bug.cgi?id=104542

**Reply by kr428** [reply]
Ah cool, thanks. Apparently https://bugs.freedesktop.org/show_bug.cgi?id=104542#c2 was what fixed it here, just installing the font didn't help. Great to see that working now. :+1:

**Reply by geckolinux** [reply]
OK good, thanks a lot for reporting and confirming the fix. I think we can include that fix out-of-the-box in a future SpiralLinux release.
